web.go

```diff
@@ -60,6 +61,7 @@ type WebUI struct {
 	params          *chaincfg.Params
 }
 
+// NewWebUI returns a pointer to a new WebUI
```

Comment:
I'd like to describe this some more.  something along the lines of "constructs a new WebUI by loading and parsing the html templates, and launching the WebSocket event handler`